src/main/java/de/maxhenkel/voicechat/api/ForgeVoicechatPlugin.java

```diff
@@ -0,0 +1,8 @@
+package de.maxhenkel.voicechat.api;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
+public @interface ForgeVoicechatPlugin {
+}
```

Comment:
Why did you add this?

Reply:
This keeps the program from crashing on non-Forge versions while doing nothing if the class is already present.

Reply:
That's not true. This is part of the API.
https://github.com/henkelmax/simple-voice-chat/blob/1.20.6/api/src/main/java/de/maxhenkel/voicechat/api/ForgeVoicechatPlugin.java

Reply:
Oh! In that case, it can be removed safely.